rm-db-format/src/main/java/org/ehrbase/openehr/dbformat/DbToRmFormat.java

```diff
@@ -211,24 +213,30 @@ private static void insertJsonEntry(ObjectNode dbRoot, DbJsonPath path, ObjectNo
             boolean isLeaf = p == leaf;
             ObjectNode child = v;
             String fieldName = p.attribute();
+            JsonNode ch = parentObject.get(fieldName);
             if (p.index() == null) {
-                JsonNode ch = parentObject.get(fieldName);
                 ObjectNode existing = (ObjectNode) ch;
-                if (existing != null) {
-                    if (isLeaf) {
-                        throw new IllegalArgumentException(
-                                "parent already has child %s (%s)".formatted(fieldName, path));
+                if (isLeaf) {
+                    if (existing != null) {
+                        /* In VersionedObjectDataStructure::handleSubObject a copy of FEEDER_AUDIT
+                         * is stored in ELEMENT, so we can treat ELEMENT as leaf.
+                         * ELEMENT/feeder_audit is replaced because it would be more complicated to skip all possible descendants
+                         */
+                        if (!FEEDER_AUDIT_ATTRIBUTE_ALIAS.equals(fieldName)) {
+                            throw new IllegalArgumentException(
+                                    "parent already has child %s (%s)".formatted(fieldName, path));
+                        }
                     }
```

Comment:
If conditions can merged into one
```suggestion
                   /* In VersionedObjectDataStructure::handleSubObject a copy of FEEDER_AUDIT
                    * is stored in ELEMENT, so we can treat ELEMENT as leaf.
                    * ELEMENT/feeder_audit is replaced because it would be more complicated to skip all possible descendants
                     */
                    if (existing != null && !FEEDER_AUDIT_ATTRIBUTE_ALIAS.equals(fieldName)) {
                        throw new IllegalArgumentException( "parent already has child %s (%s)".formatted(fieldName, path));
                    }
```